lighthouse-core/gather/gather-runner.js

```diff
@@ -158,7 +159,9 @@ class GatherRunner {
    * @return {!Promise}
    */
   static beforePass(options, gathererResults) {
-    const pass = GatherRunner.loadBlank(options.driver);
+    const blankPage = options.config.blankPage;
```

Comment:
Since blank is how we referred to this concept before and WPT also uses the 'blank' nomenclature, I'd prefer to stick with it, but open to other ideas to make it clearer what's going on here. Any thoughts?